samples/simulate-ui-login.ps1

```diff
@@ -0,0 +1,143 @@
+﻿[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$true)]
+    [string]$Appliance,
+    [Parameter(Mandatory=$true)]
+    [string]$IdentityProvider,
+    [Parameter(Mandatory=$true)]
+    [string]$UserName,
+    [Parameter(Mandatory=$true)]
+    [SecureString]$Password,
+    [Parameter(Mandatory=$false)]
+    [Switch]$Insecure = $false,
+    [Parameter(Mandatory=$true)]
+    [string]$AssetName,
+    [Parameter(Mandatory=$true)]
+    [string]$AccountName
+)
+

```

Comment:
I'd start out by setting the default error action up here to stop.  That way if something fails, they are actually able to see it.
```PowerShell
if (-not $PSBoundParameters.ContainsKey("ErrorAction")) { $ErrorActionPreference = "Stop" }
```

Reply:
Or maybe you could just simply set
```PowerShell
$ErrorActionPreference = "Stop"
```

Reply:
I went with the if block.